test/testinput/varobswriter_ukvnamelist_seviriasr.yaml

```diff
@@ -0,0 +1,106 @@
+time window:
+  begin: 2018-01-01T00:00:00Z
+  end: 2018-01-01T02:00:00Z
+
+observations:
+# This ob space shows what happens if the following are left as defaults
```

Comment:
I think you can probably remove one of the `obs space` sections in this file. The various `reject_obs...` options have been tested elsewhere. (Same in the seviriclr file.)

Reply:
I've removed one of these in each file